static/js/settings_invites.js

```diff
@@ -170,16 +167,21 @@ export function on_load_success(invites_data, initialize_event_handlers) {
             email,
             referred_by,
         };
-        const rendered_revoke_modal = render_settings_revoke_invite_modal(ctx);
-        $("#revoke_invite_modal_holder").html(rendered_revoke_modal);
-        $("#revoke_invite_modal #do_revoke_invite_button").attr("data-invite-id", meta.invite_id);
-        $("#revoke_invite_modal #do_revoke_invite_button").attr(
-            "data-is-multiuse",
-            meta.is_multiuse,
-        );
-        $("#revoke_invite_modal").modal("show");
-        $("#do_revoke_invite_button").off("click");
-        $("#do_revoke_invite_button").on("click", do_revoke_invite);
+        const modal_parent = $("#admin_invites_table");
+        const html_body = render_settings_revoke_invite_modal(ctx);
```

Comment:
Here also, why the modal parent is changed. And if there is a reason we want to use this, we can simply remove the `#revoke_invite_modal_holder` element.